src/huggingface_hub/hf_api.py

```diff
@@ -124,6 +124,7 @@
     SafetensorsParsingError,
     SafetensorsRepoMetadata,
     TensorInfo,
+    _deprecate_method,
```

Comment:
`_deprecate_method` is already imported in [line 139](https://github.com/huggingface/huggingface_hub/blob/64bcff5c7725f29514b15d4db3a7a92e3955e577/src/huggingface_hub/hf_api.py#L139)

Reply:
oh okay, that's weird that this line comes from merging main to your branch, in the current version of main, `_deprecate_method` is imported [line 139](https://github.com/huggingface/huggingface_hub/blob/64bcff5c7725f29514b15d4db3a7a92e3955e577/src/huggingface_hub/hf_api.py#L139C1-L139C50).
So, I think line 127 should be removed and the import in [`huggingface_hub/utils/__init__.py`](https://github.com/huggingface/huggingface_hub/blob/5e3aef1bda0a846e045a8144393bd0b1ffe98f56/src/huggingface_hub/utils/__init__.py#L49) as well

Reply:
> oh okay, that's weird that this line comes from merging main to your branch, in the current version of main, `_deprecate_method` is imported [line 139](https://github.com/huggingface/huggingface_hub/blob/64bcff5c7725f29514b15d4db3a7a92e3955e577/src/huggingface_hub/hf_api.py#L139C1-L139C50). So, I think line 127 should be removed and the import in [`huggingface_hub/utils/__init__.py`](https://github.com/huggingface/huggingface_hub/blob/5e3aef1bda0a846e045a8144393bd0b1ffe98f56/src/huggingface_hub/utils/__init__.py#L49) as well

Alright then, I'll do that.